csharp/src/Google.Protobuf.Test/JsonParserTest.cs

```diff
@@ -551,13 +551,9 @@ public void NumberToDouble_Valid(string jsonValue, double expectedParsedValue)
         }
 
         [Test]
-        // Skip these test cases in .NET 5 because floating point parsing supports bigger values.
-        // These big values won't throw an error in the test.
-#if !NET5_0
```

Comment:
+1 for aligning the behavior between .NET versions.
Btw the old comment wasn't fully truthful since newer .NET version dont' support bigger values, they simply return +-Inf if out of range:
https://docs.microsoft.com/en-us/dotnet/api/system.double.parse?view=net-5.0

*In .NET Core 3.0 and later, values that are too large to represent are rounded to PositiveInfinity or NegativeInfinity as required by the IEEE 754 specification. In prior versions, including .NET Framework, parsing a value that was too large to represent resulted in failure.*